src/Middleware/OutputCaching/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Don't add this file. Instead add `<InternalsVisibleTo Include="Microsoft.AspNetCore.OutputCaching.Tests" />` to the project file.